robocup-reg/web/org/models.py

```diff
@@ -39,6 +39,17 @@ class Category(models.Model):
     results = JSONField(null=True, blank=True)
     detailed_pdf = models.FileField(null=True, blank=True, upload_to="uploads/%Y/%m/%d/")
 
+    is_soccer = models.BooleanField(default=False)
```

Comment:
tieto veci ste mohli dat do vlastneho modelu napr. Soccer category...
taktiez  by som pridal `null=True, blank=True` aby to bolo kompletne, pre kategorie, co uz su vytvorene a toto nemaju